pysoa/common/transport/redis_gateway/client.py

```diff
@@ -29,15 +32,21 @@ def requests_outstanding(self):
 
     def send_request_message(self, request_id, meta, body):
         self._requests_outstanding += 1
-        meta['reply_to'] = self._receive_queue_name
+        meta['reply_to'] = '{receive_queue_name}{thread_id}'.format(
```

Comment:
Maybe some separator char between queue name and thread_id.
Let's say `'{receive_queue_name}_{thread_id}'`

Reply:
There's already a separator. The receive queue name is defined above as:

```
        self._receive_queue_name = '{send_queue_name}.{client_id}{response_queue_specifier}'.format(
            send_queue_name=self._send_queue_name,
            client_id=self.client_id,
            response_queue_specifier=BaseRedisClient.RESPONSE_QUEUE_SPECIFIER,
        )
```

And `BaseRedisClient.RESPONSE_QUEUE_SPECIFIER` is an exclamation point.